combat-trainer.lic

```diff
@@ -3077,6 +3129,17 @@ class GameState
     @construct_mode
   end
 
+  def determine_aiming_skill
+      held_types = ['Small Edged','Large Edged', 'Small Blunt','Large Blunt', 'Staves', 'Polearms']
```

Comment:
💡 Seeing these weapon skills makes me think of how weapon skills are listed in https://github.com/rpherbig/dr-scripts/blob/master/validate.lic#L600 - maybe there's a better place for that data